scripts/deploy-helm.sh

```diff
@@ -304,17 +304,24 @@ if [[ $DEPLOYMENT_TYPE == all ]] || [[ $DEPLOYMENT_TYPE == logs ]]; then
     exit 1
   fi
 
-  if [[ $USE_EXISTING_ACTIVE_GATE == true ]]; then
-    ACTIVE_GATE_CONNECTIVITY=Y
-    ACTIVE_GATE_STATE=$(curl -ksS "${DYNATRACE_LOG_INGEST_URL}/rest/health" --connect-timeout 20) || ACTIVE_GATE_CONNECTIVITY=N
-    if [[ "$ACTIVE_GATE_CONNECTIVITY" != "Y" ]]; then
-      echo -e "\e[93mWARNING: \e[37mUnable to connect to ActiveGate endpoint $DYNATRACE_LOG_INGEST_URL to check if ActiveGate is running. It can be ignored if ActiveGate host network configuration does not allow access from outside of k8s cluster."
-    fi
-    if [[ "$ACTIVE_GATE_STATE" != "RUNNING" && "$ACTIVE_GATE_CONNECTIVITY" == "Y" ]]; then
-      echo -e "\e[91mERROR: \e[37mActiveGate endpoint $DYNATRACE_LOG_INGEST_URL is not reporting RUNNING state. Please validate 'dynatraceLogIngestUrl' parameter value and ActiveGate host health."
-      exit 1
+  check_activegate_state() {
```

Comment:
wouldn't it be better to have the function at the top of the script with other functions?

Reply:
I also thought it would, but then thought it might be easier for you to diff this way and so a bit safer. I can move it up with pleasure:P